ush/ioda/bufr2ioda/run_bufr2ioda.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Naive question, but what was the issue that led to needing to keep the bash script version instead of just using the python one?

Reply:
@CoryMartin-NOAA @RussTreadon-NOAA _run_bufr2ioda.py_ requires g-w components and failed to work with my run. To test and run the python api(s), the bash version of the script is retained.

Reply:
Does it work if you have the GDASApp modulefile loaded beforehand?

Reply:
@PraveenKumar-NOAA yes.

Reply:
if that is the case, then I have a hard time envisioning why we need duplicate bash and python versions when all one needs to do is have the proper environment loaded.

Reply:
@CoryMartin-NOAA thanks, that is perfectly fine. Removed the bash version of the file.